cmd/testdata/status-pending-changes.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Comparing with the testdata with the table format, in this test it is also shown there a section for pending changes:
https://github.com/elastic/elastic-package/blob/b7ede8d5c91e0f19ce986112d810fdcba10f5206/cmd/testdata/status-pending-changes#L4

Reply:
Good one, added, thanks!